--- screen-4.0.2/etc/screenrc.screenrc	2003-12-05 13:59:39.000000000 +0000
+++ screen-4.0.2/etc/screenrc	2005-02-15 10:22:19.935843952 +0000
@@ -15,7 +15,7 @@
 startup_message off
 
 # emulate .logout message
-pow_detach_msg "Screen session of \$LOGNAME \$:cr:\$:nl:ended."
+pow_detach_msg "Screen session of $LOGNAME $:cr:$:nl:ended."
 
 # advertise hardstatus support to $TERMCAP
 # termcapinfo  * '' 'hs:ts=\E_:fs=\E\\:ds=\E_\E\\'
@@ -60,7 +60,7 @@
 # tell screen that xterm can switch to dark background and has function
 # keys.
 termcapinfo xterm 'VR=\E[?5h:VN=\E[?5l'
-termcapinfo xterm 'k1=\E[11~:k2=\E[12~:k3=\E[13~:k4=\E[14~'
+#termcapinfo xterm 'k1=\E[11~:k2=\E[12~:k3=\E[13~:k4=\E[14~'
 termcapinfo xterm 'kh=\EOH:kI=\E[2~:kD=\E[3~:kH=\EOF:kP=\E[5~:kN=\E[6~'
 
 # special xterm hardstatus: use the window title.
@@ -126,9 +126,9 @@
 # Yet another hack:
 # Prepend/append register [/] to the paste if ^a^] is pressed.
 # This lets me have autoindent mode in vi.
-register [ "\033:se noai\015a"
-register ] "\033:se ai\015a"
-bind ^] paste [.]
+#register [ "\033:se noai\015a"
+#register ] "\033:se ai\015a"
+#bind ^] paste [.]
 
 ################
 #
@@ -140,9 +140,14 @@
 # screen -t 40 2 rlogin server
 
 # caption always "%3n %t%? @%u%?%? [%h]%?%=%c"
+## alternative caption, gives window list, LOGNAME and current date:
+# caption always "%{wk}%?%-Lw%?%{bw}%n*%f %t%?(%u)%?%{wk}%?%+Lw %=%{mk}@%H %{yk}%D %{ck}%M%{wk} %{ck}%d %{gk}%c"
 # hardstatus alwaysignore
 # hardstatus alwayslastline "%Lw"
 
+# Red Hat's normal status line
+hardstatus string "[screen %n%?: %t%?] %h"
+
 # bind = resize =
 # bind + resize +1
 # bind - resize -1
